dcrtimed/dcrtimewallet/dcrtimewallet.go

```diff
@@ -30,6 +32,14 @@ type Result struct {
 	Confirmations int32
 }
 
+// BalanceResult contains information about the backing dcrwallet
+// account balance connected to by dcrtimed.
+type BalanceResult struct {
```

Comment:
Add JSON stuff.

Reply:
Fixed, same as above. This type is not serialized at the moment, but the overhead is probably negligible.

Reply:
My bad. So remove it.

Reply:
reverted.